dremio_toolkit/env_diff.py

```diff
@@ -0,0 +1,269 @@
+#########################################################################
```

Comment:
lot's of complex logic in this module(multiple loops with many conditions), I think we should at least create a functional input/output test where we create the diff snapshot and we check whether it is equal to expected snapshot. This will give us some confidence that we're not breaking anything while modifying this file. Otherwise this will quickly turn into piece of code that works but everybody is afraid to touch :P

Reply:
I will work on that.